test/e2e/framework.go

```diff
@@ -383,6 +385,71 @@ func (data *TestData) waitForCRReadyOrDeleted(timeout time.Duration, cr string,
 	return nil
 }
 
+func (data *TestData) getCRProperties(timeout time.Duration, cr_type, cr_name, namespace, key string) (string, error) {
```

Comment:
why we use cr_type, cr_name, these name are snake naming convention. We suggest unify our naming convention to camel naming convention?